README.md

```diff
@@ -111,6 +111,32 @@ make build-cli # build cli tool
 --checkpoint-storage-location-override <hplStorageFileOverride> \
 ```
 
+balances: show current on-chain balances for USDC, gas token, and any custom assets requested
+
+```shell
+./build/solvercli balances \
+  --config ./config/local/config.yml \
+  --custom-assets '{"osmosis-1":["uosmo","uion"],"celestia-1":["utia"]}'
+```
+
+inventory: show complete solver inventory including balances, settlements, and rebalances
```

Comment:
should specify that the settlements and rebalances returned here are _pending_ settlements and rebalances

Reply:
done